src/material/legacy-tabs/tab-header.spec.ts

```diff
@@ -272,9 +272,7 @@ describe('MatTabHeader', () => {
         // Focus on the last tab, expect this to be the maximum scroll distance.
         appComponent.tabHeader.focusIndex = appComponent.tabs.length - 1;
         fixture.detectChanges();
-        expect(appComponent.tabHeader.scrollDistance).toBe(
-          appComponent.tabHeader._getMaxScrollDistance(),
-        );
+        expect(appComponent.tabHeader.scrollDistance).toBe(182);
```

Comment:
Could we base these numbers on the position of the tabs relative to the viewport, rather than a hardcoded number? E.g. assert that the right edge of the tab aligns with the right edge of the viewport.

Reply:
Yes, this is a good suggestion! Actually hardcoded number didn't work well and broke some test because different browsers use a slight different styling. I will updated.